src/components/tools/Charts/Scatter/Chart.svelte

```diff
@@ -41,19 +46,66 @@
   let ymin;
   let ymax;
 
-  $: if (data) {
+  let noData = false;
+
+  $: if (Array.isArray(data) && isEmptyData(data)) {
+    noData = true;
+    xmin = new Date(Date.UTC(1950, 0, 1));
```

Comment:
Wonder if we should have these as defaults somewhere, e.g.

```js
export const DEFAULT_X_MIN = new Date(Date.UTC(1950, 0, 1));
export const DEFAULT_X_MAX = new Date(Date.UTC(2099, 0, 1));
export const DEFAULT_Y_MIN = 0;
export const DEFAULT_Y_MAX = 10;
```
Or will they vary too much between charts?

Reply:
We can add them as constants. I don't think they will vary too much between charts. Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/114/commits/8df7666d3e0ba8ae33b9018c11b10f19b9d0382b